### ISSUE TITLE
Discriminator Activation

### ISSUE THREAD

**Issue by sachinruk** [ai-tor/PyTorch-CycleGAN#8]
I couldn't help but notice that the discriminator did not have a sigmoid activation. Is there any reason for this? (https://github.com/aitorzip/PyTorch-CycleGAN/blob/67da8f9e2b69bd68763451803c7700aaccc92f18/models.py#L92)

**Comment by Einstellung** [comment]
Actually we use Fully Convolutional Networks in Discriminator, maybe you have noticed that before we use avg_pool we have this code:
`model += [nn.Conv2d(512, 1, 4, padding=1)]`
after this the output channel is 1*30*30, and next we use code
`return F.avg_pool2d(x, x.size()[2:]).view(x.size()[0], -1) `, then return channel shape is 1*1, so there is no need to use sigmoid function. We use the average pooling method instead of the fully connection layer can reduce the network size, and it helps to reduce over-fitting.
